src/TorchSharp/NN/Module.cs

```diff
@@ -722,14 +722,14 @@ public static void SaveStateDictionary(System.IO.BinaryWriter writer, Dictionary
                 {
                     writer.Encode(sd.Count); // 4 bytes
 
-                    foreach (var (k, v) in sd) {
-                        writer.Write(k);
-                        v.Save(writer);
+                    foreach (var kvp in sd) {
+                        writer.Write(kvp.Key);
+                        kvp.Value.Save(writer);
```

Comment:
I don't think you need to downgrade C#. You can explicitly set `<LangVersion>10.0</LangVersion>` and continue using new features in `netstandard2.0` target

Reply:
I am already using the latest version of the compiler. Look at the csproj to see that. I couldn't use the new syntax because the ns2.0 doesn't have the deconstructor api https://source.dot.net/#System.Private.CoreLib/KeyValuePair.cs,50.